src/screens/home/PortfolioComponent.tsx

```diff
@@ -59,33 +62,34 @@ export const PortfolioComponent = ({
             isSelected={isTotalCardSelected}
           />
         )}
-        {balances.map(
-          (
-            {
-              contractAddress,
-              symbol,
-              balance,
-            }: ITokenWithoutLogo | BitcoinNetwork,
-            i: number,
-          ) => {
-            const isSelected =
-              selectedAddress === contractAddress && !isTotalCardSelected
-            const color = isSelected
-              ? getTokenColor(symbol)
-              : sharedColors.inputInactive
-            return (
-              <PortfolioCard
-                key={i}
-                onPress={() => handleSelectedAddress(contractAddress)}
-                color={color}
-                primaryText={symbol}
-                secondaryText={balance.toString()}
-                isSelected={isSelected}
-                icon={symbol}
-              />
-            )
-          },
-        )}
+        {balances
+          .sort(sortTokensBySymbol)
+          .filter(
+            token => isDefaultToken(token.symbol) || Number(token.balance) > 0,
```

Comment:
What's the purpose of filtering here? I believe the filtering should be done when `defaultTokens` are added. What if user would want to add more tokens? it will filter it to `isDefaultToken` return value, for me it looks like it is not future proofed at all. Plus, all the tokens with 0 balance should be discarded, why?

Reply:
I believe filtering should be done specifically for RDOC and not in the components but higher app when tokens are added

Reply:
Components should just care about rendering stuff and not doing any modifications to the values unless it's for the purpose of rendering like colors etc

Reply:
> I believe filtering should be done specifically for RDOC and not in the components but higher app when tokens are added

I'll have to modify in every "call", isn't it better to keep it in one place?

Reply:
> I believe filtering should be done specifically for RDOC and not in the components but higher app when tokens are added

I don't get this part, because there is no way to user add a custom token unless receiving some amount.

Reply:
In the app those default tokens are added upon socket init or somewhere there, we can filter there and render everything as it arrives from the state

Reply:
> "RDOC should only be displayed in the token list if its value is non-zero" 

This was poorly written because if the user doesn't have a token it won''t show up at all. So, we don't need to do anything special as long as all the other tokens are shown after BTC.

Reply:
Makes sense, I removed that filter.